src/parameterizations/vertical/MOM_CVMix_KPP.F90

```diff
@@ -436,6 +437,9 @@ logical function KPP_init(paramFile, G, GV, US, diag, Time, CS, passive)
       case default ; call MOM_error(FATAL,"KPP_init: "//&
                     "Unrecognized KPP_LT_K_SHAPE option: "//trim(string))
     end select
+    call get_param(paramFile, mdl, "KPP_CVt2", CS%KPP_CVt2, &
+                   'Parameter for Stokes MOST convection entrainment', &
+                   units="nondim", default=1.6)
```

Comment:
This is in an `if (CS%LT_K_Enhancement) then` and it shouldn't be; I'm a little surprised this is passing the test suite because I don't think `CS%KPP_CVt2` is initialized before being passed to `cvmix_init_kpp()`. I didn't run a gnu debug test, which might catch it?

I'm testing a fix (moving these lines out of the if block), then I'll mark this ready to review again.